### PR TITLE
[DB sources] : Reduce CDC state compression limit to 1MB

### PR DESCRIPTION
**Pull request by akashkulk**
Context : https://airbytehq-team.slack.com/archives/C043JHEEYKG/p1708538818320589

### CODE REVIEW

File: airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialReadUtil.java
Comment (xiaohansong) [inline airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialReadUtil.java:1]: changes in this file seems unrelated

Comment (akashkulk) [reply airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialReadUtil.java:1]: Reverted. Not sure why they showed up

**Review by xiaohansong — APPROVED** [review]

**Comment by stephane-airbyte** [comment]
@akashkulk can you fix the changelog comment in the same PR where you'll use this CDK in a connector?
